### DISCUSSION TITLE
Need help with creating Microsoft.Health.Fhir.Liquid.Converter.Tool.exe

### DISCUSSION THREAD

**Discussion by yashwanth-csi** [microsoft/FHIR-Converter#403] [Q&A: Handlebars Engine]
Can anyone share a link how to create Microsoft.Health.Fhir.Liquid.Converter.Tool.exe file ... I've tried from visual studio using "C:\Users\Desktop\FHIR-Converter-main\src\Microsoft.Health.Fhir.Liquid.Converter.Tool" folder and build on the folder but still I'm getting below error : 

Process failed: Invalid command-line options.
>cd C:\Users\Desktop\FHIR-Converter-main\src\Microsoft.Health.Fhir.Liquid.Converter.Tool\bin\Debug\netcoreapp3.1

.\Microsoft.Health.Fhir.Liquid.Converter.Tool.exe convert -d \\C:\Users\yvallabhaneni\PycharmProjects\New folder\FHIR-Converter-main\FHIR-Converter-main\data\Templates\Ccda -r CCD -i \\C:\Users\yvallabhaneni\PycharmProjects\New folder\FHIR-Converter-main\FHIR-Converter-main\data\SampleData\Ccda -o \\C:\Users\yvallabhaneni\PycharmProjects\New folder\FHIR-Converter-main\FHIR-Converter-main\data\Outputfolder

Please help me with this

**Comment by zakguler** [comment]
I tried the following:
-Install Visual Studio Code
-install ext: .Net Core Tools
-Install C# for visual Studio Code
-right click on file: Microsoft.Health.Fhir.Liquid.Converter.Tool.csproj
--Build .NET Core project
--....>dotnet build "c:\zak_dev\z_MS_Convert_tool\FHIR-Converter-main\src\Microsoft.Health.Fhir.Liquid.Converter.Tool\Microsoft.Health.Fhir.Liquid.Converter.Tool.csproj"
this will create the .exe file in the bin directory.

**Comment by granadacoder** [comment]
1.  See "My Notes" here : https://github.com/microsoft/FHIR-Converter/discussions/475

2.  Try taking out the space after your -argument flags.  (-d, -r, etc). AND... try a simpler path with no spaces in the foldernames (note your "New folder"). Command line tools and spaces-in-file-names-or-path-names are not friends.

After you get a "no space" foldername/filenames working.. you can circle back and try single or double quotes around them.

But start simple, then work out the syntax sugar... is my in-general advice.
